public/css/screen.css

```diff
@@ -1,14 +1,12 @@
 @charset "UTF-8";
-/*------------------------------------*    $BASE VARIABLES
-\*------------------------------------*/
-/**
- * Theme colors
- */
 /*
 *************************************************
 
 Screen Styles
```

Comment:
@markhuot Should we remove `screen.scss` from the repo? This would automatically be generated when the user runs the command `grunt`